### PR TITLE
Feat: allow users to change dotenv file with env var

### PR DESCRIPTION
**Pull request by JoaoCostaIFG**
This facilitates the usage of docker secrets for the keys/tokens.

For example on a `docker-compose.yml` file:

```yaml
muse:
  image: codetheweb/muse:latest
  container_name: muse
  secrets:
    - muse_env
  environment:
    - ENV_FILE=/run/secrets/muse_env
  restart: unless-stopped
  
secrets:
  muse_env:
    file: /usr/local/etc/muse_env
```

See [relevant dotenv doc](https://www.npmjs.com/package/dotenv#path)

- [ ] I updated the changelog

### CODE REVIEW

**Comment by JoaoCostaIFG** [comment]
Not sure if I should update the changelog. Please let me know.

**Comment by codetheweb** [comment]
Cool!

1. Let's default `ENV_FILE` to `/config` in the Dockerfile so there's a sensible default.
2. We should also document this in the README.

Thank you!

**Comment by Codixer** [comment]
@JoaoCostaIFG 
Do you still want this PR to be processed?

**Comment by JoaoCostaIFG** [comment]
> @JoaoCostaIFG Do you still want this PR to be processed?

I haven't been using discord, so I completely forgot about this PR. Sorry about that.
I just pushed the requested changes.
Let me know if there's anything else that needs work.

**Comment by codetheweb** [comment]
#### :package: :robot: A new release has been made for this pull request.

To play around with this PR, pull `ghcr.io/museofficial/muse:pr-1040`.

Images are available for x86_64 and ARM64.

> Latest commit: 55a1ce664821eb1405cb07d8c26f65b80722aa5c

File: Dockerfile
Comment (Codixer) [inline Dockerfile:1]: @JoaoCostaIFG Could you fix the legacy key perchance? See comment/warning by the Actions

**Review by Codixer — CHANGES_REQUESTED** [review]
See review

File: Dockerfile
Comment (JoaoCostaIFG) [reply Dockerfile:1]: Doné :)

Comment (Codixer) [reply Dockerfile:1]: Thanks!

**Comment by Codixer** [comment]
LGTM

**Review by Codixer — APPROVED** [review]

**Comment by codetheweb** [comment]
🚀 Released in [Release v2.9.5](https://github.com/museofficial/muse/releases/tag/v2.9.5).

**Pull request merged**